packages/apps/crm/src/screens/contact/ContactDetailsScreen.js

```diff
@@ -44,8 +50,10 @@ const ContactDetailsScreen = ({route}) => {
   }, [dispatch, idContact]);
 
   useEffect(() => {
-    getContactAPI();
-  }, [getContactAPI]);
+    if (isFocused) {
```

Comment:
You need to also block return of the screen to prevent refresh in sub-component.
Adding `!isFocused` in if condition before return part is working